__init__.py

```diff
@@ -262,6 +247,28 @@ def perform_get_instruction_low_level_il(self, data, addr, il):
             il.append(il.unimplemented())
             return 2
 
+    def perform_is_never_branch_patch_available(self, data, addr):
+        ins = self._get_instruction(data, addr)
+        return self._is_conditional_branch(ins)
+
+    def perform_is_always_branch_patch_available(self, data, addr):
+        ins = self._get_instruction(data, addr)
+        return self._is_conditional_branch(ins)
+
+    def perform_always_branch(self, data, addr):
+        ins = self._get_instruction(data, addr)
+        dst = ins._operands[0]
+        v = (dst.immediate_value - 2) / 2
+        v = (v & 0xFFF) | 0xc000
+        return struct.pack('<H', v)
+
+    def perform_never_branch(self, data, addr):
+        return "\x00\x00"
+
+    def perform_convert_to_nop(self, data, addr):
+        return "\x00\x00"
+
+
```

Comment:
One last thing, remove this additional newline here, then we should be good :+1: